.github/workflows/azure-saml-ses-deploy-test.yaml

```diff
@@ -0,0 +1,46 @@
+name: azure_saml_ses deploy test
+on:
+  workflow_dispatch: {}
+  pull_request:
+    branches: main
+    paths:
+      - 'cloud/**'
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+
+    env:
+      ARM_TENANT_ID: ${{ secrets.AZURE_SERVICE_PRINCIPAL_TENANT }}
+      ARM_SUBSCRIPTION_ID: ${{ secrets.AZURE_SUBSCRIPTION_ID }}
+      ARM_CLIENT_ID: ${{ secrets.AZURE_SERVICE_PRINCIPAL_ID }}
+      ARM_CLIENT_SECRET: ${{ secrets.AZURE_SERVICE_PRINCIPAL_PASSWORD }}
+      AWS_ACCESS_KEY_ID: ${{ secrets.ACCESS_KEY_ID }}
+      AWS_SECRET_ACCESS_KEY: ${{ secrets.SECRET_ACCESS_KEY }}
+    steps:
+      - name: check out branch
+        uses: actions/checkout@v2
+
+      - name: login to Azure
+        run: cloud/azure/bin/cli-login
+
+      - name: resolve latest snapshot tag
+        run: source cloud/shared/bin/resolve-latest-snapshot-tag
+
+      - name: copy the staging file
+        run: |
+          curl https://raw.githubusercontent.com/civiform/staging-azure-deploy/main/civiform_config.sh > civiform_config.sh
```

Comment:
suggestion (blocking): please comment why we do this

Reply:
@goldblatt one more here

Reply:
huh wait i added a comment might have gotten reverted

Reply:
ahh yeah it should have it now, maybe github had a lil delay